packages/loader/container-definitions/src/error.ts

```diff
@@ -10,30 +10,23 @@ export enum ContainerErrorType {
     /**
      * Some error, most likely an exception caught by runtime and propagated to container as critical error
      */
-    genericError = "genericError",
+    genericError = "container.genericError",
```

Comment:
i'm not really follow the advantage of prefixing there. won't it just make it harder to query

Reply:
I think it will be helpful to be able to group by "layer" where an error originated.  If you want to cross-cut and do all `dataIntegrityError`, just do `| where errorType has "dataIntegrityError"`